jvb/src/main/kotlin/org/jitsi/videobridge/Endpoint.kt

```diff
@@ -287,6 +289,8 @@ class Endpoint @JvmOverloads constructor(
         setupDtlsTransport()
 
         conference.videobridge.statistics.totalEndpoints.incrementAndGet()
+
+        logger.info("$id source names: $isUsingSourceNames")
```

Comment:
Can you log something like `Created new endpoint isUsingSourceNames=$isUsingSourceNames, iceControlling=$iceControlling` (the ID is already in the logger context)